presto-hive/pom.xml

```diff
@@ -148,6 +148,12 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.jodd</groupId>
+            <artifactId>jodd-core</artifactId>
+            <version>3.6.6</version>
```

Comment:
All versions should go in the `dependencyManagement` section in the root pom, even if it is only used in one module.